node/node.go

```diff
@@ -1374,9 +1375,13 @@ func (app *App) Start(ctx context.Context) error {
 	}
 
 	if app.Config.MetricsPush != "" {
-		metrics.StartPushingMetrics(app.Config.MetricsPush,
-			app.Config.MetricsPushUser, app.Config.MetricsPushPass, app.Config.MetricsPushPeriod,
-			app.host.ID().String(), app.Config.Genesis.GenesisID().ShortString())
+		metrics.StartPushingMetrics(
+			app.Config.MetricsPush,
+			app.Config.MetricsPushUser,
+			app.Config.MetricsPushPass,
+			app.Config.MetricsPushHeader,
+			app.Config.MetricsPushPeriod,
+			app.host.ID().String()[:5], app.Config.Genesis.GenesisID().ShortString())
```

Comment:
what do you think about hashing the first 5 char?

Reply:
maybe hash all then and use 5 chars? i can do

Reply:
actually it doesn't hide anything, if i want i can hash all identities found in the network and match them

Reply:
yes. but at least its a step towards anomynizing. somebody have to pay the computation to get it....

Reply:
But the same info will be publicly available in the epoch when ATX will be published, right? So why are we trying to hide it?

Reply:
this is p2p id